src/mainwindow.cpp

```diff
@@ -76,6 +76,17 @@ MainWindow::MainWindow(TerminalConfig &cfg,
 
     setupUi(this);
 
+    // Allow insane small sizes - reason:
+    // https://github.com/lxqt/qterminal/issues/181 - Minimum size
+    // https://github.com/lxqt/qterminal/issues/263 - Decrease minimal height
+    QFontMetrics metrics(Properties::Instance()->font);
+#if (QT_VERSION >= QT_VERSION_CHECK(5,11,0))
+    int spaceWidth = metrics.horizontalAdvance(QChar(QChar::Space));
+#else
+    int spaceWidth = metrics.width(QChar(QChar::Space));
+#endif
+    setMinimumSize(QSize(10 * spaceWidth, metrics.height()));
```

Comment:
, 2 * metrics. would be better - prevent follow up bugs

Reply:
otherwise fine

Reply:
It would be too "big". My measures have changed since our last discussion: https://github.com/lxqt/qterminal/pull/611

Really... it can be made very small without vanishing or creating any problem.

Reply:
ok, convinced